example/mysql_user/create_mysql.sql

```diff
@@ -0,0 +1,17 @@
+
+-- DDL for object User.
+CREATE TABLE `user` (
+  `user_id` BIGINT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(255) NOT NULL DEFAULT 'DefaultName',
+  `phone` VARCHAR(20) NOT NULL,
```

Comment:
Explicitly declare the default data value for all `NOT NULL` column, to avoid unexpected situations when the default values (aka: zero values) of MySQL and Go are different.

- [Go - Zero Values](https://tour.golang.org/basics/12)
- [MySQL :: Data Type Default Values](https://dev.mysql.com/doc/refman/5.7/en/data-type-defaults.html)